identity/webapp/src/app.ts

```diff
@@ -49,7 +56,31 @@ export async function createApp(router: TypedRouter<any, any>): Promise<Koa> {
   app.use(json({ pretty: process.env.NODE_ENV !== 'production' }));
   app.use(logger());
   app.use(errorHandler);
-  app.use(router.routes()).use(router.allowedMethods());
+
+  const koaRouter = router.router;
+  const secured = (ctx, next) => {
```

Comment:
nit: would call this `isAuthenticated` or something like that as the meaning of "secured" is ambiguous